src/toolbar_gui.cpp

```diff
@@ -728,10 +728,10 @@ static CallBackFunction MenuClickIndustry(int index)
 
 static void ToolbarVehicleClick(Window *w, VehicleType veh)
 {
-	int dis = ~0;
+	int dis = 0;
```

Comment:
give you use `SetBit`, I would feel better if this would be an unsigned something :)